src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -233,3 +234,122 @@ def basic(
         self.logger.info("done creating tasks")
         helper.wait()
         return helper.job
+
+    def merge(
+        self,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: str,
+        *,
+        target_exe: File = File("fuzz.exe"),
+        setup_dir: Optional[Directory] = None,
+        inputs: Optional[Directory] = None,
+        output_container: Optional[Container] = None,
+        reboot_after_setup: bool = False,
+        duration: int = 24,
+        target_options: Optional[List[str]] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        check_retry_count: Optional[int] = None,
+        crash_report_timeout: Optional[int] = None,
+        tags: Optional[Dict[str, str]] = None,
+        wait_for_running: bool = False,
+        wait_for_files: Optional[List[ContainerType]] = None,
+        extra_files: Optional[List[File]] = None,
+        existing_inputs: Optional[List[Container]] = None,
+        dryrun: bool = False,
+        notification_config: Optional[NotificationConfig] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        overwrite_output_container: bool = False,
```

Comment:
I think this should change to emphasize that we will _delete missing_ from the remote.

Reply:
(cc @chkeita)

Reply:
@ranweiler  is **delete_missing** you suggestion for the new name ?

Reply:
Perhaps 'minimize_result_set'

Reply:
How about, 'preserve_existing_outputs' and flip the logic

Reply:
I like `preserve_existing_results` or `keep_redundant_results`. Or to flip it back to the original polarity, `delete_redundant_results`?

Reply:
I think the default should be to delete,and give the option to preserve.

Reply:
Agreed. I guess if this ends up as a flag arg on the CLI, it would look like `--keep_redundant_results`, with a default of `false`. If we did `--delete_redundant_results` with a default of `true`, then disabling it would have to be specified as `--delete_redundant_results=false`. So maybe `--keep_..` is a better UX, given the desired default.

Reply:
(I'm happy to go with whatever you two decide on, just wanted us to put our heads together on the name to emphasize that something may get _deleted_ in some cases, not just mutated)

Reply:
it looks like we have two votes (me and brian) for `preserve_existing_results` and one (joe) for `keep_redundant_results`.
I will go ahead and rename to `preserve_existing_results`